src/System.ObjectModel/src/System/Collections/ObjectModel/ObservableCollection.cs

```diff
@@ -312,17 +697,24 @@ protected void CheckReentrancy()
         //------------------------------------------------------
 
         #region Private Methods
+
         /// <summary>
-        /// Helper to raise a PropertyChanged event for the Count property
+        /// Helper function to determine if a collection contains any elements.
         /// </summary>
-        private void OnCountPropertyChanged()
+        /// <param name="collection">The collection to evaluate.</param>
+        /// <returns></returns>
+        private static bool ContainsAny(IEnumerable<T> collection)
+        {
+            using (IEnumerator<T> enumerator = collection.GetEnumerator())
+                return enumerator.MoveNext();
+        }
+
+        private void OnEssentialPropertiesChanged()
         {
             OnPropertyChanged(EventArgsCache.CountPropertyChanged);
+            OnIndexerPropertyChanged();
         }
 
-        /// <summary>
-        /// Helper to raise a PropertyChanged event for the Indexer property
-        /// </summary>
```

Comment:
Why did you delete the comment?

Reply:
Probably mistake during copy-paste. [Fixed](https://github.com/dotnet/corefx/pull/26482/files#diff-1ac8157faff4154cd7e79ef80e4d3103R737).